### PR TITLE
Zk02

### PR DESCRIPTION
**Pull request by mmzaid**

### CODE REVIEW

File: src/main/java/com/labeasy/controller/CityController.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import com.labeasy.dto.CityDto;
+import com.labeasy.services.MasterService;
+
+@Controller
+@RequestMapping("/master")
+public class CityController {
+	private final MasterService masterService;
+
+	public CityController(MasterService masterService) {
```
Comment (Nasruddinkhan) [inline src/main/java/com/labeasy/controller/CityController.java:26]: Add constructor autowired

File: src/main/java/com/labeasy/controller/PinCodeController.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import com.labeasy.dto.PinCodeDto;
+import com.labeasy.services.MasterService;
+
+@Controller
+@RequestMapping("/master")
+public class PinCodeController {
+	
+	private final MasterService masterService;
+	
+	public PinCodeController(MasterService masterService) {
```
Comment (Nasruddinkhan) [inline src/main/java/com/labeasy/controller/PinCodeController.java:23]: Add constructor autowired

File: src/main/java/com/labeasy/entity/City.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	@Column(name = "city_id")
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "cityseq")
+	@SequenceGenerator(name = "cityseq", sequenceName = "city_seq", allocationSize = 1, initialValue = 1)
+	private Long cityId;
+	@Column(name = "city_name", length = 30)
+	private String cityName;
+	@ManyToOne
+	@JoinColumn(name = "state_id", nullable = false)
+	private State state;
+	@OneToMany(mappedBy = "city",cascade = CascadeType.ALL)
```
Comment (Nasruddinkhan) [inline src/main/java/com/labeasy/entity/City.java]: Add lazy intilization

**Review by Nasruddinkhan — CHANGES_REQUESTED** [review]
Added few comments please check

**Comment by mmzaid** [comment]
added missing autowired anotation and lazy loding in city module

**Review by Nasruddinkhan — APPROVED** [review]

**Pull request merged**